games/devtest/mods/unittests/misc.lua

```diff
@@ -363,3 +363,10 @@ local function test_sandbox()
 	assert(debug.getinfo(function() end, "f").func ~= nil)
 end
 unittests.register("test_sandbox", test_sandbox)
+
+local function test_str_pack_unpack()
+	local fmt = "d"
+	assert(fmt:packsize() == 8)
+	assert(fmt:unpack(fmt:pack(42.3)) == 42.3)
```

Comment:
a more complex test won't hurt
```suggestion
	local fmt = "> d!2 xh"
	assert(fmt:packsize() == 12) -- 8 double, 1 padding, 1 implicit align, 2 short
	local r = fmt:unpack(fmt:pack(42.3, -384)) 
	assert(r[1] == 42.3 and r[2] == -384)
```